ethereum/deploy/12-erc20-app.ts

```diff
@@ -35,15 +35,21 @@ module.exports = async ({
       }
     ],
     libraries: {
-        ScaleCodec: scaleCodecLibrary.address
+      ScaleCodec: scaleCodecLibrary.address
     },
     log: true,
     autoMine: true,
   });
 
   await deployments.deploy("TestToken", {
     from: deployer,
-    args:["Test Token", "TEST"],
+    args: ["Test Token", "TEST"],
+    log: true,
+  });
+
+  await deployments.deploy("TestToken721Enumerable", {
```

Comment:
i think this should go in /ethereum/deploy/13-erc721-app.ts rather?

Reply:
could you add the loop to mint some nfts to the deployer account too in 13-erc721-app.ts  as well?

Reply:
I'd rather leave minting and any other custom setup for the dashboard to be done in a separate hardhat script that can be run after the deployments when required. For two reasons:
1) The deployment script already takes a long time to complete, especially with geth.
2) At least for E2E testing purposes, our ERC20 and ERC721 tests are already doing the minting without relying on the deployment script (See https://github.com/Snowfork/snowbridge/blob/main/test/test/erc721app.js#L32).